### PR TITLE
KiCad.KiCad version 5.1.12_1

### PR DESCRIPTION
**Pull request by swoga**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/35025)

### CODE REVIEW

File: manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://kicad-downloads.s3.cern.ch/windows/stable/kicad-5.1.12_1-x86_64.exe
```
Comment (ItzLevvie) [inline manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:15]:
```suggestion
  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Fk%2Fki%2Fkicad%2Fkicad-5.1.12_1-x86_64.exe
```

File: manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  InstallerSha256: 993C02F1B593964014D639AABD5A847CCBB697EE6BF3BA0602FF470861511340
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://kicad-downloads.s3.cern.ch/windows/stable/kicad-5.1.12_1-i686.exe
```
Comment (ItzLevvie) [inline manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:22]:
```suggestion
  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Fk%2Fki%2Fkicad%2Fkicad-5.1.12_1-i686.exe
```

Comment (ItzLevvie) [reply manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:22]: Worldwide link so people in e.g. China don't get slow download speeds from EU mirror.

Validation-Domain needs to be taken a look by an engineer.

File: manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:15]: Worldwide link so people in e.g. China don't get slow download speeds from EU mirror.

Validation-Domain needs to be taken a look by an engineer.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml (hunk 1 above)
Comment (swoga) [reply manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:15]: I was unable to download the full version from this mirror, therefore I choose the next best source.

File: manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml (hunk 2 above)
Comment (swoga) [reply manifests/k/KiCad/KiCad/5.1.12_1/KiCad.KiCad.installer.yaml:22]: I was unable to download the full version from this mirror, therefore I choose the next best source.

**Pull request closed without merging**